waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/impl/PrefixBasedDatabaseMappingService.java

```diff
@@ -258,17 +310,21 @@ public PanopticOperationHandler getPanopticOperationHandler() {
       public List<TableMeta> getTableMeta(String db_patterns, String tbl_patterns, List<String> tbl_types) {
         Map<DatabaseMapping, String> databaseMappingsForPattern = databaseMappingsByDbPattern(db_patterns);
 
-        BiFunction<TableMeta, DatabaseMapping, Boolean> filter = (tableMeta, mapping) -> isWhitelisted(
-            mapping.getDatabasePrefix(), tableMeta.getDbName());
-
+        BiFunction<TableMeta, DatabaseMapping, Boolean> filter = (tableMeta, mapping) -> {
```

Comment:
Minor, but I generally think if the code on the RHS of a `->` is more than a line or two then it probably warrants being extracted into its own method. It won't make that much difference here since there's nothing else in this method anyway. Up to you whether you want to leave it like this.

Reply:
Yeah, probably better have a separate method.